cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -22,6 +22,7 @@ def self.config(env)
     whitelisted_cookies = [
       'hour_of_code',
       'language_',
+      'partner',
```

Comment:
I'm about to bump the version for another http_cache change with https://github.com/code-dot-org/code-dot-org/pull/5876

@wjordan if I merge in my PR after Dave's will that take care of both changes then? So we don't have to bump it twice?